README.md

```diff
@@ -257,6 +260,26 @@ and `.dsc` files, do this:
 
 Note that we no longer actively try to get it into the Debian repository.
 
+Building the Windows Installer
+---------------------------
+
+You will need a Debian-ish operating system, and the following packages:
+
+```
+apt-get install build-essentials gcc-mingw-w64-i686 libgcrypt-mingw-w64-dev \
```

Comment:
Sorry I was unclear, the question was asking whether it should be using Debian packages that don't have "w64" in the package name so that you can build on a non-64bit Linux machine, but now that I think about it, the packages have the same name on an i386 Debian too